.github/dependabot.yaml

```diff
@@ -0,0 +1,39 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/drivers/
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/jobs/apollo-src/
+  - package-ecosystem: 'docker'
+    directory: .openshift-ci/jobs/integration-tests/
+  - package-ecosystem: 'docker'
+    directory: collector/connscrape-test/
+  - package-ecosystem: 'docker'
+    directory: collector/container/
```

Comment:
We have 2 dockerfiles here:

    Dockerfile.template uses ARGs to set the image being used as base and I'm not sure dependabot will pick up on it.
    Dockefile.ubi is not being used right now, so updating it would not really have any effect.

Reply:
It should https://github.com/dependabot/feedback/issues/145

Reply:
That link talks about dependabot finding files with names other than `Dockerfile`, I'm more concerned about the actual content of the dockerfile: https://github.com/stackrox/collector/blob/3433fd8211380a99ac418c2c5616c5de2830b76a/collector/container/Dockerfile.template#L1-L5

Reply:
I give it a test and you are right https://github.com/janisz/collector/pulls

Reply:
If Dockefile.ubi is not being used right now, why not deleting it?

Reply:
Sorry for the delayed response. Basically, that image was used as a mirror to the downstream release, it fell out of use because OSCI is similar enough to downstream and we didn't bother to re-implement it. But now we are considering going to GHA, so it might make sense to go back to building this image.